### ISSUE TITLE
Assume Role does not work for NetApp

### ISSUE THREAD

**Issue by bryanheo** [NetApp/terraform-provider-netapp-cloudmanager#122]
Hello

We are trying to deploy Connector and CVO via assume role as shown below but it returns error.
When we deploy the connector and CVO with IAM user, the deployment works fine but if we use assume role, it returns 'Error: UnauthorizedOperation'. The both IAM user and Assume role use the same policy
Can NetApp resources be deployed by Assume role rather than IAM user? For the security reason, we prefer the assume role.

```
provider "aws" {
  region = var.region

  assume_role {
    role_arn     = "arn:aws:iam::${var.account_id}:role/tfe-netapp-deploy"
    session_name = "tfe-netapp"
  }

}
```

Error
![Screenshot 2022-08-17 at 21 41 06](https://user-images.githubusercontent.com/12592663/185238988-160209b0-3183-4c48-ba10-a89980e16523.png)

Regards
Moon

**Comment by lonico** [comment]
Yes, this is not supported at present.  This is something we are currently working on for FSxN.

**Comment by bryanheo** [comment]
@lonico Thank you for the confirmation.
We will use IAM user for now

**Comment by lonico** [comment]
DEVOPS-5351
